### PR TITLE
[docs] add header template for jaeger receiver and exporter.

### LINKED ISSUE #10155: Add status header for jaeger

### LINKED ISSUE #10156: Add status header for jaeger

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
hi,I add header template for jaeger receiver and exporter, pls review :). I don't know how to change the two links above the template to the correct ones.

Fixes #10155
Fixes #10156

### CODE REVIEW

File: exporter/jaegerexporter/README.md [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+
+| Status                   |         |
+| ------------------------ |---------|
+| Stability                | beta    |
+| Supported pipeline types | traces  |
+| Distributions            | contrib |
+
+<README contents>
+
+[beta]:https://github.com/open-telemetry/opentelemetry-collector-contrib#beta
```
Comment (codeboten) [inline exporter/jaegerexporter/README.md:12]:
```suggestion
[beta]:https://github.com/open-telemetry/opentelemetry-collector#beta
```

**Comment by codeboten** [comment]
Thanks for the contribution @Frapschen! Just a couple of comments.

File: exporter/jaegerexporter/README.md [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+| Status                   |               |
+| ------------------------ |---------------|
+| Stability                | beta          |
+| Supported pipeline types | traces        |
+| Distributions            | core, contrib |
+
+<README contents>
+
+[beta]:https://github.com/open-telemetry/opentelemetry-collector#beta
+[contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
```
Comment (codeboten) [inline exporter/jaegerexporter/README.md:13]: Please move these links to the bottom of the readme

File: exporter/jaegerexporter/README.md [hunk 3]
```diff
@@ -1,10 +1,20 @@
 # Jaeger gRPC Exporter
 
+
+| Status                   |               |
+| ------------------------ |---------------|
+| Stability                | beta          |
+| Supported pipeline types | traces        |
+| Distributions            | core, contrib |
+
+<README contents>
```
Comment (codeboten) [inline exporter/jaegerexporter/README.md:10]: this should be removed

**Comment by Frapschen** [comment]
probably everything is done.

**Review by codeboten — APPROVED** [review]

**Pull request merged**